modules/microshift-install-rhel-image-mode-conc.adoc

```diff
@@ -29,7 +29,8 @@ To use image mode for {op-system-base}, ensure that the following resources are
 
 The workflow for using image mode with {microshift-short} includes the following steps:
 
-. Build the {microshift-short} bootc image.
+. Find and use the {microshift-short} container images to install {op-system-base-full}.
+. Build the custom {microshift-short} bootc image.
```

Comment:
- It's either find and use, or build
- Publish the image is related to the build case only